Lib/importlib/_bootstrap.py

```diff
@@ -960,10 +960,14 @@ def _find_and_load_unlocked(name, import_):
 def _find_and_load(name, import_):
     """Find and load the module."""
     with _ModuleLockManager(name):
+        # bpo-30891: don't use "try/except KeyError" for performance.
+        # __import__() already starts by checking for sys.modules[name],
+        # _find_and_load() is usually called when the module is not in
+        # sys.modules.
         if name not in sys.modules:
             return _find_and_load_unlocked(name, import_)
+        module = sys.modules[name]
```

Comment:
This is the branch that I think could use a comment to say "Another thread updated sys.modules while this one was waiting for the import lock"